src/core/metrics.py

```diff
@@ -78,6 +78,71 @@ def get_models_helper() -> List[str]:
         return []
 
 
+def get_namespaces_helper() -> List[str]:
```

Comment:
Why not name it `get_namespaces` as this is what the function returns. There's no helper here.